File: software/bindings/csharp/ExampleBeep.cs
using Tinkerforge;

class Example
{
	private static string HOST = "localhost";
	private static int PORT = 4223;
	private static string UID = "ABC"; // Change to your UID

	static void Main() 
	{
		IPConnection ipcon = new IPConnection(HOST, PORT); // Create connection to brickd
		BrickletPiezoBuzzer pb = new BrickletPiezoBuzzer(UID); // Create device object
		ipcon.AddDevice(pb); // Add device to ip connection
		// Don't use device before it is added to a connection


		// Make 2 second beep
		pb.Beep(2000);

		System.Console.WriteLine("Press ctrl+c to exit");
		ipcon.JoinThread();
    }
}

File: software/examples/vbnet/ExampleBeep.vb
Imports System
Imports Tinkerforge

Module ExampleBeep
    Const HOST As String = "localhost"
    Const PORT As Integer = 4223
    Const UID As String = "XYZ" ' Change XYZ to the UID of your Piezo Buzzer Bricklet

    Sub Main()
        Dim ipcon As New IPConnection() ' Create IP connection
        Dim pb As New BrickletPiezoBuzzer(UID, ipcon) ' Create device object

        ipcon.Connect(HOST, PORT) ' Connect to brickd
        ' Don't use device before ipcon is connected

        ' Make 2 second beep
        pb.Beep(2000)

        Console.WriteLine("Press key to exit")
        Console.ReadLine()
        ipcon.Disconnect()
    End Sub
End Module
